app/models/manageiq/providers/openstack/storage_manager/swift_manager/cloud_object_store_container.rb

```diff
@@ -31,6 +34,8 @@ def self.params_for_create(ems)
   end
 
   def self.raw_cloud_object_store_container_create(ext_management_system, options)
+    cloud_tenant_id = options.delete(:cloud_tenant_id)
+    cloud_tenant = CloudTenant.find_by(:id => cloud_tenant_id) if cloud_tenant_id
     cloud_tenant = options.delete(:cloud_tenant)
```

Comment:
@GilbertCherrie should this line be deleted since you're setting `cloud_tenant` one line above?

Reply:
I think I remember leaving a line like this in there for supporting both the old ui-classic controller as well as the api.

If anything, please move this line above the one using `cloud_tenant_id` for sure.
Not sure if we need to support both of them during the transition.

Also, could you find a way to get more of the logic for extracting the `cloud_tenant_id` from the original data and move it into here?

Reply:
Yeah if you actually want to have both we need to only set this if it isn't already so `cloud_tenant ||= ...` but I'd prefer only having one way of doing this if at all possible